packages/adp-tooling/test/unit/preview/adp-preview.test.ts

```diff
@@ -7,12 +7,20 @@ import type { SuperTest, Test } from 'supertest';
 import supertest from 'supertest';
 import express from 'express';
 import { readFileSync, existsSync } from 'fs';
+import { renderFile } from 'ejs';
 
 interface GetFragmentsResponse {
     fragments: { fragmentName: string }[];
     message: string;
 }
 
+interface GetControllersResponse {
+    controllers: { controllerName: string }[];
+    message: string;
+}
+
+jest.mock('ejs');
```

Comment:
Do you really need to mock it? Wouldn't his be a good opportunity to also test that the generation works i.e. path correct etc.

Reply:
Changed in: https://github.com/SAP/open-ux-tools/pull/1275/commits/dbded1d478b30424a53ef16dc6720a5fd654354a